src/main/java/org/sagebionetworks/web/client/presenter/SettingsPresenter.java

```diff
@@ -255,6 +260,32 @@ public void configure() {
     view.setOauthClientSettingsVisible(
       DisplayUtils.isInTestWebsite(ginInjector.getCookieProvider())
     );
+
+    if (DisplayUtils.isInTestWebsite(ginInjector.getCookieProvider())) {
+      view.setTwoFactorAuthSettingsVisible(true);
+    } else {
+      // If 2FA is enabled, show the settings pane, even if experimental mode is disabled
+      // When released from experimental mode (SWC-6411), this block can be removed
+      jsClient
+        .getTwoFactorAuthStatusForCurrentUser()
+        .addCallback(
+          new FutureCallback<TwoFactorAuthStatus>() {
+            @Override
+            public void onSuccess(TwoFactorAuthStatus result) {
+              view.setTwoFactorAuthSettingsVisible(
+                TwoFactorState.ENABLED.equals(result.getStatus())
+              );
+            }
+
+            @Override
+            public void onFailure(Throwable t) {
+              GWT.log("Unable to get 2FA status", t);
```

Comment:
Not worried about the error handling because this is temporary code that will be removed when 2FA comes out of experimental mode